go/pkg/sdk/sdk_test.go

```diff
@@ -259,3 +404,126 @@ func TestSdkGetSupportedStablecoins(t *testing.T) {
 		})
 	}
 }
+
+func printUserBalance(from common.Address, client *ethclient.Client) {
+	balance, err := client.BalanceAt(context.Background(), from, nil)
+	if err != nil {
+		log.Fatalf("Failed to get the balance: %v", err)
+	}
+	ethBalance := new(big.Float).Quo(new(big.Float).SetInt(balance), big.NewFloat(1e18))
+	fmt.Printf("Eth balance of %s: %f ETH\n", from.Hex(), ethBalance)
+}
+
+func getAssetUserBalance(from common.Address, asset common.Address, client *ethclient.Client, chainId int64, isDeposit bool) (*big.Int, error) {
```

Comment:
to be refactor: we should not pass "isDeposit" but should just pass the asset address. it should appear clearly on the test file if we're retrieving DAI/sDAI/USDC balance